rapidfuzz/details/common.hpp

```diff
@@ -345,52 +345,55 @@ struct CharSet {
 
 template <typename T>
 struct MatrixVectorView {
-    explicit MatrixVectorView(T* vector, int64_t cols) : m_vector(vector), m_cols(cols)
+    explicit MatrixVectorView(T* vector, std::size_t cols) : m_vector(vector), m_cols(cols)
     {}
 
-    T& operator[](int64_t col)
+    T& operator[](std::size_t col)
     {
         assert(col < m_cols);
         return m_vector[col];
     }
 
-    int64_t size() const
+    std::size_t size() const noexcept
     {
         return m_cols;
     }
 
 private:
     T* m_vector;
-    int64_t m_cols;
+    std::size_t m_cols;
 };
 
 template <typename T>
 struct ConstMatrixVectorView {
-    explicit ConstMatrixVectorView(const T* vector, int64_t cols) : m_vector(vector), m_cols(cols)
+    explicit ConstMatrixVectorView(const T* vector, std::size_t cols) : m_vector(vector), m_cols(cols)
```

Comment:
Both in `MatrixVectorView` and `ConstMatrixVectorView` I forgot to remove the explicit keyword, when I added the cols argument. In addition the constructors and the `operator[]` of `ConstMatrixVectorView` can be noexcept